src/styles/Home.css

```diff
@@ -0,0 +1,29 @@
+@media screen and (min-width: 378px) {
+  .city-container .link-wrapper .city-card {
+    padding: 30px 10px;
+  }
+
+  .city-container .link-wrapper:nth-child(4n+2),
+  .city-container .link-wrapper:nth-child(4n+3) {
+    background-color: #2f85c7ef;
+  }
+
+  .weather-info li {
+    font-family: 'Roboto Slab', serif;
+    color: whitesmoke;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .city-container .link-wrapper .city-card {
+    padding: 30px 15px;
+  }
+
+  .city-container .link-wrapper:nth-child(odd) {
+    background-color: #468bd0e3;
+  }
+
+  .city-container .link-wrapper:nth-child(even) {
+    background-color: #2f85c7ef;
+  }
```

Comment:
- Great job for making your page responsive, I think it would be better to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
    - Including:
          Typography: font face, size, and weight
          Layout: composition and space between elements. to fit the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) 👍 

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/HtetWaiYan7191/weather-app/assets/52010789/d1dfa0f7-e8c6-4e0f-8648-6939a1168534" width="200">|<img src="https://github.com/HtetWaiYan7191/weather-app/assets/52010789/38a5d7c2-0d58-460a-80d8-de6c51293c9c" width="200">|


😎️

🤝️

Reply:
Thank you very for showing me sir. I will change my min-width to 375 px .